yt_dlp/extractor/parti.py

```diff
@@ -0,0 +1,113 @@
+
+from .common import InfoExtractor
+from ..utils import UserNotLive, int_or_none, parse_iso8601, url_or_none, urljoin
+from ..utils.traversal import traverse_obj
+
+
+class PartiBaseIE(InfoExtractor):
+    def _call_api(self, path, video_id, note=None):
+        return self._download_json(
+            f'https://api-backend.parti.com/parti_v2/profile/{path}', video_id, note)
+
+
+class PartiVideoIE(PartiBaseIE):
+    IE_NAME = 'parti:video'
+    _VALID_URL = r'https?://(?:www\.)?parti\.com/video/(?P<id>\d+)'
+    _TESTS = [
+        {
+            'url': 'https://parti.com/video/66284',
+            'info_dict': {
+                'id': '66284',
+                'ext': 'mp4',
+                'title': 'NOW LIVE ',
+                'upload_date': '20250327',
+                'is_live': False,
+                'categories': ['Gaming'],
+                'thumbnail': 'https://assets.parti.com/351424_eb9e5250-2821-484a-9c5f-ca99aa666c87.png',
+                'channel': 'ItZTMGG',
+                'timestamp': 1743044379,
+            },
+            'params': {'skip_download': 'm3u8'},
+        },
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        data = self._call_api(f'get_livestream_channel_info/recent/{video_id}', video_id)
+        formats = self._extract_m3u8_formats(
+            urljoin('https://watch.parti.com', data['livestream_recording']), video_id, 'mp4')
+
+        return {
+            'id': video_id,
+            'formats': formats,
+            'is_live': False,
+            **traverse_obj(data, {
+                'title': ('event_title', {str}),
+                'channel': ('user_name', {str}),
+                'thumbnail': ('event_file', {url_or_none}),
+                'categories': ('category_name', {str}, all),
```

Comment:
`filter` before `all` to filter out empty strings. You can also `filter` after `all` to filter out empty lists, but it's not necessary
```suggestion
                'categories': ('category_name', {str}, filter, all),
```